src/sql/src/plan/statement/ddl.rs

```diff
@@ -1141,11 +1153,26 @@ fn get_encoding_inner<T: sql_parser::ast::AstInfo>(
         }
         Format::Protobuf(schema) => match schema {
             ProtobufSchema::Csr {
-                csr_connector: CsrConnectorProto { seed, .. },
+                csr_connector:
+                    CsrConnectorProto {
+                        url,
+                        seed,
+                        with_options: ccsr_options,
+                    },
             } => {
                 if let Some(CsrSeedCompiledOrLegacy::Compiled(CsrSeedCompiled { key, value })) =
                     seed
                 {
+                    let mut ccsr_with_options = normalize::options(&ccsr_options);
+
+                    // We validate here instead of in purification, to match the behavior of avro
```

Comment:
which avro behavior exactly is this matching?

Reply:
validating that there are no empty options, even though the protobuf decoder doesn't use this ccsr config